go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -766,21 +766,25 @@ func (bp *backupPipe) HashString() string {
 	return hex.EncodeToString(bp.crc32.Sum(nil))
 }
 
-func (bp *backupPipe) ReportProgress(period time.Duration, logger logutil.Logger) {
+func (bp *backupPipe) ReportProgress(period time.Duration, logger logutil.Logger, restore bool) {
+	messageStr := "restoring "
```

Comment:
Nit, but log messages most often use sentence capitalization (and punctuation if multiple sentences) — including the ones we're replacing. So I'd prefer capitalizing the first letter of the first word here too. Obviously very minor though.

Reply:
That would have been my preference too. However, we are using the same string once in the middle and once at the beginning. If you look at the logs pasted in the issue, we have something like "Done taking Backup", which is arguably worse than "done taking backup".
I didn't want to create two strings, one capitalized and one not, and it didn't seem worth fiddling with functions in the experimental text/cases package, so I chose what felt like the lesser evil.

Reply:
I did change `completed` to `Completed`.